tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -53,11 +53,14 @@ SELECT add_continuous_aggregate_policy('mat_m1', '1 day'::interval, 10 );
 ERROR:  function add_continuous_aggregate_policy(unknown, interval, integer) does not exist at character 8
 SELECT add_continuous_aggregate_policy('mat_m1', 10, '1 day'::interval, '1 h'::interval);
 ERROR:  invalid parameter value for end_interval
+--start_interval < end_interval
+SELECT add_continuous_aggregate_policy('mat_m1', 5, 10, '1h'::interval) as job_id \gset
+ERROR:  start interval should be greater than end interval
 SELECT add_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval) as job_id \gset
 --adding again should warn/error
 SELECT add_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval, if_not_exists=>false);
 ERROR:  refresh policy already exists for continuous aggregate "mat_m1"
-SELECT add_continuous_aggregate_policy('mat_m1', 10, 20, '1h'::interval, if_not_exists=>true);
+SELECT add_continuous_aggregate_policy('mat_m1', 20, 15, '1h'::interval, if_not_exists=>true);
```

Comment:
I don't see a point in this change. Should the error be different?

Reply:
This is to make sure we have a test case for this scenario (line 64). Added tests for start < end.

Reply:
Should it be added at another place, since the error here is that the policy already exists? Or do I miss anything?

Reply:
Sorry, I misunderstood. I still often confused that the start interval should be bigger than the end interval. Thank you for explaining, now I follow.